esmf_regrid/experimental/unstructured_scheme.py

```diff
@@ -413,21 +415,23 @@ def __init__(
 
     def __call__(self, cube):
         """
-        Regrid this cube onto the target grid of this regridder instance.
+        Regrid this :class:`~iris.cube.Cube` onto the target grid of this regridder instance.
 
-        The given cube must be defined with the same mesh as the source
-        cube used to create this MeshToGridESMFRegridder instance.
+        The given :class:`~iris.cube.Cube` must be defined with the same mesh as the source
+        cube used to create this :class:`MeshToGridESMFRegridder` instance.
 
         Parameters
         ----------
-        cube : cube
-            A iris.cube.Cube instance to be regridded.
+        cube : :class:`iris.cube.Cube`
+            A :class:`~iris.cube.Cube` instance to be regridded.
 
         Returns
         -------
-            A cube defined with the horizontal dimensions of the target
-            and the other dimensions from this cube. The data values of
-            this cube will be converted to values on the new grid using
+        :class:`iris.cube.Cube`
+            A :class:`~iris.cube.Cube` defined with the horizontal dimensions of the target
+            and the other dimensions from this :class:`~iris.cube.Cube`. The
```

Comment:
I think this should refer to the argument "cube" rather than the class "Cube".

Reply:
99dd4241559692dd5174d53618288c0c5af1c514